packages/next/src/server/app-render/work-async-storage.external.ts

```diff
@@ -73,6 +73,7 @@ export interface WorkStore {
 
   rootParams: Params
   dynamicIOEnabled: boolean
+  dev: boolean
```

Comment:
We really should have a global constant that represents the dev runtime (as opposed to the NODE_ENV). this works for now I suppose but this is such a critical context useful in so many places tying it to an ALS that may not even be in scope is frustratingly limiting.

Reply:
not that this should block landing this PR. but maybe you want to tackle that in aother effort if not now